src/test/scala/chiselTests/InlineSpec.scala

```diff
@@ -0,0 +1,124 @@
+// See LICENSE for license details.
+
+package chiselTests
+
+import chisel3._
+import chisel3.util.{InlineInstance, InlineHelpers}
+import firrtl.FirrtlExecutionSuccess
+import firrtl.passes.InlineAnnotation
+import firrtl.{ir => fir}
+import firrtl.WDefInstance
+import firrtl.Mappers._
+import org.scalatest.{FreeSpec, Matchers}
+
+class InlineSpec extends FreeSpec with ChiselRunners with Matchers {
+
+  trait Internals { this: Module =>
+    val io = IO(new Bundle{ val a = Input(Bool()) })
+  }
+
+  class Foo extends Module with Internals with InlineHelpers with InlineInstance
+  class Bar extends Module with Internals with InlineHelpers
+
+  def collectInstances(c: fir.Circuit): Array[WDefInstance] = {
```

Comment:
Why Array? It's not all that important but usually we just use `Seq`

Reply:
Was due to the internal use of a mutable `ArrayBuffer`, but that gets converted automatically. I'll switch the internal to a `ListBuffer` as that's all that needed and return a `Seq`. Consistency is good...